src/System.Data.SqlClient/src/System/Data/SqlClient/SqlDbColumn.cs

```diff
@@ -0,0 +1,94 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+
+using System.Data.Common;
+
+namespace System.Data.SqlClient
+{
+    internal class SqlDbColumn : DbColumn
+    {
+        private bool _isBrowseModeInfoConsumed;
+        private _SqlMetaData _metadata;
+
+        internal SqlDbColumn(_SqlMetaData md, bool isBrowseModeInfoConsumed)
+        {
+            this._metadata = md;
+            this._isBrowseModeInfoConsumed = isBrowseModeInfoConsumed;
+        }
+
+        private void populate()
```

Comment:
Or just get rid of the `Populate` method altogether and move this code into the constructor.